site/docs/v1/tech/apis/_import-users-request-body.adoc

```diff
@@ -102,8 +102,11 @@ The User's mobile phone number. This is useful is you will be sending push notif
 The User's password. If [field]#encryptionScheme# is provided this value is the existing password hash, if [field]#encryptionScheme# is not provided this value is assumed to be plain text and it will be hashed during import.
 +
 When importing a bcrypt hash, you may have a value such as `$2a$10$N9qo8uLOickgx2ZMRZoMyeIjZAgcfl7p92ldGxad68LJZdL17lhWy`. This single string represents the bcrypt version, factor, salt and hash.
+The first 22 characters after the final `$` value is the salt and the last 31 characters are the hash.
 +
-[field]#password# should contain only the password hash. For the above bcrypt hash, `N9qo8uLOickgx2ZMRZoMyeIjZAgcfl7p92ldGxad68LJZdL17lhWy` is the hash value and should be placed in the [field]#password# field.
+[field]#password# should contain only the password hash.
+For the above bcrypt hash, `N9qo8uLOickgx2ZMRZoMye` is the salt and should be placed in the [field]#salt# field.
+`IjZAgcfl7p92ldGxad68LJZdL17lhWy` is the hash value and should be placed in the [field]#password# field.
```

Comment:
We could also document the format such as `[version][factor][hash [0 - 21 salt][22 - password]]`.

Reply:
Great call. Probably a good blog post. I added some info about the factor.